corehq/apps/accounting/bootstrap/utils.py

```diff
@@ -127,9 +126,9 @@ def _ensure_product_rate(monthly_fee, edition, verbose, apps):
     try:
         # TODO - remove after squashing migrations
         SoftwareProduct = apps.get_model('accounting', 'SoftwareProduct')
-        product = SoftwareProduct(name=product_name, product_type=SoftwareProductType.COMMCARE)
+        product = SoftwareProduct(name=product_name, product_type='CommCare')
         if hasattr(product, 'product_type'):
-            product.product_type = SoftwareProductType.COMMCARE
+            product.product_type = 'CommCare'
```

Comment:
What do lines 130 and 131 do? It looks like product_type should always already be 'CommCare'

Reply:
Missed this redundancy - removed the if statement